internal-registry/eclipse/openshift-cloud-terminal/4.5.0/meta.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+publisher: eclipse
+name: openshift-cloud-terminal
+version: 4.5.0
+type: Che Editor
+displayName: OpenShift Cloud Terminal
+title: OpenShift Cloud Terminal
+description: OpenShift Cloud Terminal provides an ability to use terminal inside OpenShift Console.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/eclipse/che-machine-exec/
+firstPublicationDate: "2020-05-13"
+category: Other
+spec:
+  endpoints:
+   -  name: cloud-shell
+      public: true
+      targetPort: 4444
+      attributes:
+        protocol: http
+        type: ide
+        path: /static/
+        discoverable: false
+        secure: true
+        cookiesAuthEnabled: true
+  containers:
+   - name: che-machine-exec
+     image: "quay.io/eclipse/che-machine-exec:nightly"
```

Comment:
later it will be replaced with a released che-machine-exec version that is compatible with openshift4.5 or even maybe image built for openshift4.5

Reply:
I really hope you don't plan to have two different che-machine-exec branches... one for 4.4 and earlier, and another for 4.5+. 

Once 4.5 is out, CRW support plan is basically 4.5+, so moving upstream to 4.5+ too is likely OK as well.